tests/run_covid19_cpu_gpu_consistency_checks.py

```diff
@@ -4,18 +4,39 @@
 # For full license text, see the LICENSE file in the repo root
 # or https://opensource.org/licenses/BSD-3-Clause
 
-import os
+"""
+Consistency tests for comparing the cuda (gpu) / no cuda (cpu) version
+"""
+
+import GPUtil
+
+try:
+    num_gpus_available = len(GPUtil.getAvailable())
+    assert num_gpus_available > 0, "The env consistency checker needs a GPU to run!"
+    print(
+        f"Inside env_cpu_gpu_consistency_checker.py: "
+        f"{num_gpus_available} GPUs are available."
+    )
+    from warp_drive.utils.env_registrar import EnvironmentRegistrar
+    from warp_drive.env_cpu_gpu_consistency_checker import EnvironmentCPUvsGPU
+except ModuleNotFoundError:
+    raise ModuleNotFoundError(
+        "The env consistency checker requires the 'WarpDrive' package, please run "
+        "'pip install rl-warp-drive' first."
+    ) from None
+except ValueError:
+    raise ValueError("The env consistency checker needs a GPU to run!") from None
 
-from warp_drive.utils.env_registrar import EnvironmentRegistrar
+import os
 
-from ai_economist.foundation.env_cpu_gpu_consistency_checker import EnvironmentCPUvsGPU
+from ai_economist.foundation.env_wrapper import FoundationEnvWrapper
 from ai_economist.foundation.scenarios.covid19.covid19_env import (
     CovidAndEconomyEnvironment,
 )
 
-env_registrar = EnvironmentRegistrar()
+env_registry = EnvironmentRegistrar()
```

Comment:
Why is it called registry if the class is called Registrar? Would be good to make it consistent